### PR TITLE
Test definition from repo

### PR DESCRIPTION
**Pull request by maierlars**
https://github.com/arangodb/arangodb/pull/16052

### CODE REVIEW

File: scripts/runTests.ps1 [hunk 1]
```diff
@@ -12,79 +12,85 @@ Function global:registerSingleTests()
...
-    # Note that we intentionally do not register the hot_backup test here,
-    # since it is currently not supported on Windows. The reason is that
-    # the testing framework does not support automatic restarts of instances
-    # and hot_backup currently needs a server restart on a restore operation
-    # on Windows. On Linux and Mac we use an exec operation for this to
-    # restart without changing the PID, which is not possible on Windows.
-    # registerTest -testname "hot_backup"
+    if (Test-Path "$INNERWORKDIR/ArangoDB/tests/test-definitions.txt" -PathType Leaf) {
+        Write-Host "Using test definitions from repo..."
+        C:\Python310\python.exe $INNERWORKDIR/ArangoDB/scripts/generateJenkinsScripts.py $INNERWORKDIR/ArangoDB/tests/test-definitions.txt -f ps1 | Invoke-Expression
```
Comment (dothebart) [inline scripts/runTests.ps1:17]: using absolute paths here is a receipy to disaster. other parts of powershell oskar lean on python being in the path; this should be done here as well.

**Review by dothebart — APPROVED** [review]
Please move the generator script in this repo in `jenkins/helper` and remove the absolute paths from the python interpreter invocation.

Please create the backports of https://github.com/arangodb/arangodb/pull/16052

**Comment by dothebart** [comment]
tiny note:
the current state of these scripts abandon `registerTest -testname "agency" -weight 2 -sniff true` if I get that correctly.

File: scripts/runTests.ps1 (hunk 1 above)
Comment (maierlars) [reply scripts/runTests.ps1:17]: Neither `python` nor `python3` worked. I tried it, as you can see in commit [7ee8c13](https://github.com/arangodb/oskar/pull/386/commits/7ee8c13198c13cf0692db248fd87b009445cc630) and [88d10e4](https://github.com/arangodb/oskar/pull/386/commits/88d10e44e131e0a6013295997bf85dd23ce861f6).

**Pull request merged**